win32/select.c

```diff
@@ -87,7 +87,7 @@ PHPAPI int php_select(int max_fd, fd_set *rfds, fd_set *wfds, fd_set *efds, stru
 
 	if (n_handles == 0) {
 		/* plain sockets only - let winsock handle the whole thing */
-		return select(max_fd, rfds, wfds, efds, tv);
+		return select(NULL, rfds, wfds, efds, tv);
```

Comment:
@weltling The `nfds` parameter is always ignored, according to [Microsoft’s documentation](https://msdn.microsoft.com/en-us/library/windows/desktop/ms740141(v=vs.85).aspx).